kolibri/plugins/learn/assets/src/views/content-card-carousel/index.vue

```diff
@@ -213,25 +249,41 @@
   // width of card + gutter
   $card-height = 210px
 
+
   .content-carousel
     margin-top: 1em
-    margin-bottom: 1em
     clearfix()
 
     &-details
+      // vertical-align: bottom
+      // text-align: justify
       clearfix()
+      position: relative
+      margin-bottom: 1em
 
       &-header
         float: left
-        text-align: left
-        margin-bottom: 1em
-        h2
-          margin: 0
-      &-view-all
-        float: right
-        color: white
-        background-color: $core-action-normal
+        text-decoration: none
+        color: $core-text-default
 
+      &-view-more
+        position: absolute
+        right: 0
+        bottom: 0
+
+      &-link-icon
+        // puts drops the center angle of the arrow to about the level of text's midline
+        transform: translateY(30%)
+        display: inline
+
+        ../-header &
```

Comment:
It goes up one scope (to `&-view-more`), retrieves that selector, and adds `-header`. Then the `&` brings in the current scope's selector.

So, final selector in this case would be `.content-carousel-details-view-more content-carousel-details-link-icon` because `&` exists in the next scope up and the scope above that. I needed the `../` because the icon is different between the header and the `view-more` link.

I see what you mean about it being complicated. Pulling the header logic out of this will simplify things. `&` is a common feature across nearly all preprocessors, though. So I'll keep that if that's alright with you.